src/main/_docs/use-che-as-an-ide/ide-debug.md

```diff
@@ -71,3 +71,116 @@ It is also possible to add this command as a Dockerfile instruction for a custom
 FROM eclipse/cpp_gcc
 RUN echo  "set remotetimeout 10" > /home/user/.gdbinit
 ```
+
+# PHP
+
+The PHP and Zend stacks come with the Zend Debugger module installed and configured. Debugging both PHP CLI scripts and PHP web apps is supported.
+
+The debugging workflow involves the following steps:
+
+1. Launch the Zend Debugger Client to start listening for new debug sessions.
+1. Optionally set breakpoints in the PHP editor.
+1. Trigger a debug session from the CLI script or the web app.
+
+## Starting the Zend Debugger Client
+
+Eclipse Che has the Zend Debugger Client integrated in the Web IDE. For launching the Zend Debugger Client:
+
+1. Go to `Run > Edit Debug Configurations` from the main menu.
+1. Create new `PHP` configuration.
+1. Change any settings if necessary. The defaults are usually OK.
+1. Click the `Debug` button.
+
+![php-debug-configuration.png]({{ base }}/docs/assets/imgs/php-debug-configuration.png)
```

Comment:
Should change to `![php-debug-configuration.png]({{base}}{{site.links["php-debug-configuration.png"]}})`